src/compiler/crystal/codegen/codegen.cr

```diff
@@ -69,8 +69,10 @@ module Crystal
       end
     end
 
-    def codegen(node, single_module = false, debug = Debug::Default)
-      visitor = CodeGenVisitor.new self, node, single_module: single_module, debug: debug
+    def codegen(node, single_module = false, frame_pointers = FramePointers::Auto,
+                debug = Debug::Default)
```

Comment:
**issue:** Method calls to the previous signature can keep working if we append the additional parameter at the end:

```suggestion
    def codegen(node, single_module = false, debug = Debug::Default,
                frame_pointers = FramePointers::Auto)
```